cpp/include/Ice/Communicator.h

```diff
@@ -65,19 +61,15 @@ namespace Ice
         /// @see #shutdown
         void waitForShutdownAsync(std::function<void()> completed) noexcept;
 
-        /// Check whether communicator has been shut down.
-        /// @return `true` if the communicator has been shut down; `false` otherwise.
+        /// Checks whether this communicator has been shut down.
+        /// @return `true` if the communicator has been shut down, `false` otherwise.
         /// @see #shutdown
         [[nodiscard]] bool isShutdown() const noexcept;
 
-        /// Convert a stringified proxy into a proxy.
-        /// For example, <code>MyCategory/MyObject:tcp -h some_host -p 10000</code> creates a proxy that refers to the
-        /// Ice object having an identity with a name "MyObject" and a category "MyCategory", with the server running on
-        /// host "some_host", port 10000. If the stringified proxy does not parse correctly, the operation throws
```

Comment:
As a matter of style, the summary/description should never include "if you pass a bad param, this function throws Xyz". The correct way to document such behavior is in `@throws`.